products/nativescript/tns.py

```diff
@@ -40,7 +40,7 @@ def exec_command(command, cwd=Settings.TEST_RUN_HOME, platform=Platform.NONE, em
         :param uglify: If true pass `--env.uglify` to command.
         :param snapshot: If true pass `--env.snapshot` to command.
         :param log_trace: If not None pass `--log <level>` to command.
-        :param just_launch: If true pass `--justlaunch` to command.
+        :param just_launch: If true pass `--just_launch` to command.
```

Comment:
May be mistake by me could you please fix it "--justlaunch"

Reply:
Fixed